src/server/views/prototypes/mot-test/review.njk

```diff
@@ -0,0 +1,367 @@
+{% extends 'layouts/mts.layout.njk' %}
+
+{% set pageTitle = 'Defect Categories' %}
+
+{% set pageBreadcrumbItems = [
+  { text: 'Home', url: '/prototypes' },
+  { text: 'MOT test results', url: '/prototypes/mot-test' },
+  { text: 'MOT test summary' }
+] %}
+
+{% block content %}
+
+  {% call components.gridRow() %}
+
+    {% call components.columnFull() %}
+
+      {{ components.pageHeading(heading='MOT test summary') }}
+
+      <section class='dvsa-summary'>
+
+        <div class='dvsa-summary__section '>
+
+          <div class='dvsa-summary__row'>
+
+            <div class='dvsa-summary__col dvsa-summary__col--one-third'>
+
+              <h2 class='dvsa-summary__row-heading'>
+                Test information
+
+                <span class='dvsa-summary__test-result dvsa-summary__test-result--pass'>Pass</span>
+              </h2>
+
+            </div>
+
+            <div class='dvsa-summary__col dvsa-summary__col--one-third'>
+
+              <dl class='dvsa-summary__data-list'>
+
+                <span class='dvsa-summary__group dvsa-summary__group--first'>
```

Comment:
Why do you have a span wrapper around ``dt`` and ``dd``?

Reply:
To control spacing in multiple dt, dd groups.

Reply:
Agreed to control spacing via DL rather than within DL